modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsMXBeanImpl.java

```diff
@@ -47,13 +54,37 @@
     /** Cluster metrics update mutex. */
     private final Object clusterMetricsMux = new Object();
 
+    /** Topology version metric. */
+    private final LongMetric topVer;
```

Comment:
Let's remove this metric from new framework.
"CurrentTopologyVersion" would be added by the #8145 in the "io.discovery" registry.

Reply:
Done.